extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -475,19 +471,27 @@ class MongoQueueProcessor {
                 zenkoObjMd);
 
             const objVal = sourceEntry.getValue();
-            // Always call putObject with version params undefined so
-            // that mongoClient will use putObjectNoVer which just puts
-            // the object without further manipulation/actions.
-            // S3 takes care of the versioning logic so consuming the queue
-            // is sufficient to replay the version logic in the consumer.
-            return this._mongoClient.putObject(bucket, key, objVal, undefined,
+            const params = { usePHD: true };
```

Comment:
Minor: seems a bit cleaner to leave this `undefined` for the `putObjectNoVer` condition, and assign it as an object with those fields for the `putObjectVerCase4`.